README.md

```diff
@@ -0,0 +1,64 @@
+# Diplomat
+
+Diplomat is a conversational agent framework for goal-oriented group discussion. Recent work in human-computer interaction has explored the use of conversational agents as facilitators for group goal-oriented discussions. Inspired by this work and by the apparent lack of tooling available to support it, we created Diplomat, a Python-based framework for building conversational agent facilitators. Diplomat is designed to support simple specification of agent functionality as well as customizable integration with online chat services. We document a preliminary user study we conducted to help inform the design of Diplomat.
+
+## How to run Diplomat? 
+Diplomat has currently been configured to run seamlessly with Slack. 
+
+#### TLDR:
+Setup your python environment by running 
+`pip3 install requirements.txt`.
+
+You need the following to run an instance of diplomat: 
+1. Configuration file containing information of which features to run. `(conf.json)`
+2. A Slackbot token generated by Slack.
+3. Name of the channel you want Diplomat to monitor group discussions in.
+4. User ID of your bot. 
+
+Once you have these, simply run diplomat by using the following command: 
+`python3 slack_integration -p conf.json -t <slackbot-token> -c <channel-id> -b <bot-id>` 
+
+### Detailed steps for a developing your custom plugin:
```

Comment:
The title here is a little misleading, because the contents of this section also include subsections on Slack setup.